theme/templates/resource-landing-page/modals.html

```diff
@@ -266,14 +266,13 @@ <h4 class="modal-title" id="edit-creator"> Edit Author</h4>
                         <div class="form-group">
                             <fieldset>
                                 {# NAME #}
-                                <div v-if="is_person" class="control-group">
+                                <div class="control-group">
                                     <label for="creator-name" class="control-label requiredField">Name<span
```

Comment:
just to confirm this was intentional, in the UI now, control-group class will be applied in all cases

Reply:
@cuahsimarko I'll keep this in mind as well, but I did not make any change in terms of control-group class. I only removed v-if condition for is_person check in this case.